File: app/src/main/java/id/co/gitsolution/androidchallege/help/HelpActivity.java
package id.co.gitsolution.androidchallege.help;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

import id.co.gitsolution.androidchallege.Constant;
import id.co.gitsolution.androidchallege.R;
import id.co.gitsolution.androidchallege.model.HelpAdapter;

public class HelpActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_help);

        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.rec_view);

        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        recyclerView.setAdapter(new HelpAdapter(Constant.getHelp()));
    }
}

File: app/src/main/java/id/co/gitsolution/androidchallege/help/HelpActivity.kt
package id.co.gitsolution.androidchallege.help

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import id.co.gitsolution.androidchallege.Constant
import id.co.gitsolution.androidchallege.R
import id.co.gitsolution.androidchallege.global.GlobalAdapter
import kotlinx.android.synthetic.main.activity_help.*

class HelpActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_help)

        rec_view.layoutManager = LinearLayoutManager(this)
        rec_view.adapter = object: GlobalAdapter(R.layout.list_item_help, Constant.getHelp()){}
    }
}
